betatest/admin.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
```

Comment:
Done in https://github.com/open-craft/opencraft/pull/6/commits/cb0cf5df710789a5eedbb12eca33bfd1a8d54a38 . Updating the year only takes running a sed command, so I've left it, and changed it to `2015-2016` in all files.